composer.json

```diff
@@ -13,7 +13,8 @@
         "druidfi/omen": "^0.2.0",
         "drupal/core-composer-scaffold": "^9.0",
         "drupal/core-recommended": "^9.0",
-        "drupal/hdbt": "~1.0",
+        "drupal/hdbt": "1.x-dev",
```

Comment:
Gin theme is required, but it's missing from composer files.

Reply:
It's all good. The HDBT Admin theme has Gin theme as a dependency in composer.json and there was a problem with local environment composer caches. `composer install --no-cache` solved the problem for me.